ansible-ipi-install/roles/installer/tasks/03_extract_installer.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+- name: Copy PullSecret into {{ pullsecret_file }}
+  copy:
+    content: "{{ pullsecret }}"
+    dest: "{{ pullsecret_file }}"
+    owner: "{{ myuser }}"
+    group: "{{ myuser }}"
+    mode: '0644'
+
+- name: Extracting the installer
```

Comment:
Is there a reason not to put the installer in /usr/local/bin?

Reply:
I'm just running it how we had it in our install guide. In that guide, we only put the oc binary in /usr/local/bin